packages/app/src/runner/event-manager.ts

```diff
@@ -338,13 +338,8 @@ export class EventManager {
   setup (config) {
     Cypress = this.Cypress = this.$CypressDriver.create(config)
 
-    // expose Cypress globally
```

Comment:
this comment needs to remain, since it was originally expressing what `window.Cypress = Cypress` does